File: Python/Fibonacci.py
while True:
	n = int(input("Type the index of the Fibonacci series you want to find: "))
	if n <= 0: print("The index you type must be a positive integer, please try again\n")
	else: break
if n == 1: c = 1
if n == 2: c = 1
else:
	a = 1
	b = 1
	for i in range(3, n + 1):
		c = a + b
		a = b
		b = c
print(f"The index of {n} of the Fibonacci series is the number {c}")

File: Website/Web Programming Bootcamp/day-9/part-2/exercise-3/fibonacci.js
function fibonacci(n) {
    let a = 0;
    let b = 1;
    if (n === 1) return a;
    else if (n === 2) return b;
    else for (let i = 3; i <= n; i++) {
        var c = a + b;
        a = b;
        b = c;
    }
    return c;
}

let n = prompt("Enter the number: ");
console.log(fibonacci(n));